apps/google-analytics-4/frontend/src/components/config-screen/GoogleAnalyticsConfigPage.tsx

```diff
@@ -25,9 +25,14 @@ export default function GoogleAnalyticsConfigPage() {
     {} as Partial<EditorInterface>
   );
   const [originalParameters, setOriginalParameters] = useState<KeyValueMap>({});
+  const [hasServiceCheckErrors, setHasServiceCheckErrors] = useState<boolean>(true);
```

Comment:
philosophical question: does it make sense for the default state to be that the app _has_ errors?

Reply:
Good question -> I think your philosophical question is more direct in this use case: Because there's so much hidden error logic in the children components, I decided to be cautious and say `You're wrong until proven right` rather than `You're right until you're proven wrong`.

However, if you mean philosophical in the meta sense where we determine abstract universal rules - I got no freaking clue tbh. I see merits for both